docs/examples/tutorials/token_bridge_contract/contracts/aztec/nft/src/nft.nr

```diff
@@ -0,0 +1,26 @@
+// docs:start:nft_note_struct
+use dep::aztec::{
+    macros::notes::note,
+    protocol_types::{
+        address::AztecAddress,
+        traits::Packable,
+    },
+};
+
+#[derive(Eq, Packable)]
+#[note]
+pub struct NFTNote {
+    owner: AztecAddress,
+    randomness: Field,
+    token_id: Field,
+}
+// docs:end:nft_note_struct
+
+// docs:start:nft_note_new
+impl NFTNote {
+    pub fn new(owner: AztecAddress, token_id: Field) -> Self {
+        // The randomness preserves privacy by preventing brute-forcing
+        NFTNote { owner, randomness: unsafe { dep::aztec::oracle::random::random() }, token_id }
```

Comment:
should we put all imports at the top of the file? feels cleaner :slightly_smiling_face:

Reply:
sure!